MANIFEST.in

```diff
@@ -0,0 +1 @@
+exclude tests/**
```

Comment:
Is this just excluding for the package?

Reply:
yes, so the unit tests are not included in the package